crates/eip7928/src/block_access_list.rs

```diff
@@ -0,0 +1,13 @@
+//! Contains the [`BlockAccessList`] type, which represents a simple list of account changes.
+
+use crate::account_changes::AccountChanges;
+use alloc::vec::Vec;
+
+/// This struct is used to store `account_changes` in a block.
+pub type BlockAccessList = Vec<AccountChanges>;
+
+/// Computes the hash of the given block access list.
+#[cfg(feature = "rlp")]
+pub fn compute_block_access_list_hash(bal: &BlockAccessList) -> alloy_primitives::B256 {
```

Comment:
accepting a slice is better in this case because this doesn't lock in the Vec
```suggestion
pub fn compute_block_access_list_hash(bal: &[AccountChanges]) -> alloy_primitives::B256 {
```